### ISSUE TITLE
how to confirm  the  hyper-parameter ` [2, 4, 5, 9, 10, 11, 13]` ,` [1, 3, 6, 7, 8, 12, 14]`

### ISSUE THREAD

**Issue by dreamercv** [pjl1995/CTracker#4]
some code `for seq_num in [2, 4, 5, 9, 10, 11, 13]:` ,`for seq_num in [1, 3, 6, 7, 8, 12, 14]:`in the file `test.py` .how to confirm the  hyper-parameter `**[2, 4, 5, 9, 10, 11, 13]**,
look forward to hearing from you!

**Comment by pjl1995** [comment]
This hyper-parameter represents the image sequence in MOT17 dataset. [2, 4, 5, 9, 10, 11, 13] means the training set and [1, 3, 6, 7, 8, 12, 14] means the test set.
You can refer to this page:  https://motchallenge.net/data/MOT17Det/

**Comment by dreamercv** [comment]
can you upload daidu cloud disk!  think you!
